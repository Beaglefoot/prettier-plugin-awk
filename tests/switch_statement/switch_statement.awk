{
      switch(x+1) {
    case 1:
      print "hi"
      break
    case 2:
    default:
      typeof(x);
  }
}
