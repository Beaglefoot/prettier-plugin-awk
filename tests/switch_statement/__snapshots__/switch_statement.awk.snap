{
      switch(x+1) {
    case 1:
      print "hi"
      break
    case 2:
    default:
      typeof(x);
  }
}
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
{
    switch (x + 1) {
        case 1:
            print "hi"
        case 2:
        default:
            typeof(x)
    }
}
